pkg/apis/ignite/v1alpha2/types.go

```diff
@@ -202,6 +202,29 @@ type VMNetworkSpec struct {
 	Ports meta.PortMappings `json:"ports,omitempty"`
 }
 
+// VMStorageSpec defines the VM's Volumes and VolumeMounts
+type VMStorageSpec struct {
+	Volumes      []Volume      `json:"volumes,omitempty"`
+	VolumeMounts []VolumeMount `json:"volumeMounts,omitempty"`
+}
+
+// Volume defines named storage volume
+type Volume struct {
+	Name        string             `json:"name"`
+	BlockDevice *BlockDeviceVolume `json:"blockDevice,omitempty"`
+}
+
+// BlockDeviceVolume defines a block device on the host
+type BlockDeviceVolume struct {
+	Path string `json:"path"`
```

Comment:
Add validation that this field must be set, and must be absolute

Reply:
Done in the validation added in ca75db1.